File: app/src/main/java/com/thibaudperso/sonycamera/sdk/model/PictureResponse.java
package com.thibaudperso.sonycamera.sdk.model;

import com.thibaudperso.sonycamera.sdk.CameraWS;

import java.io.Serializable;


public class PictureResponse implements Serializable {

    public CameraWS.ResponseCode status;
    public long time; // picture received time in millis
    public String url;

    @Override
    public String toString() {
        return "PictureResponse{" +
                "status=" + status +
                ", time=" + time +
                ", url='" + url + '\'' +
                '}';
    }
}

File: app/src/main/java/com/thibaudperso/sonycamera/sdk/model/PictureResponse.kt
package com.thibaudperso.sonycamera.sdk.model

import com.thibaudperso.sonycamera.sdk.ResponseCode
import java.io.Serializable

data class PictureResponse(var status: ResponseCode? = null,
                           var receivedTimeMillis: Long = 0,
                           var url: String? = null) : Serializable
